website/src/pages/en/token-api/token-api-faq.mdx

```diff
@@ -0,0 +1,120 @@
+  - [Response Format & Data Handling](#response-format--data-handling)
+  - [Common Issues & Troubleshooting](#common-issues--troubleshooting)
+  - [Integration & Usage](#integration--usage)
+
+## General
+
+### What blockchains does the Token API support?
+
+Currently, the Token API supports Ethereum, Binance Smart Chain (BSC), Polygon, Optimism, Base, and Arbitrum One.
+
+### Why isn't my API key from thegraph.market working?
+
+Make sure you are using the Access Token, not the API key. You can generate an access token from the dashboard on thegraph.market, using the dropdown menu next to each API key.
+
+### How current is the data provided by the API relative to the blockchain?
+
+The API provides data up to the Last Irreversible Block (LIB).
+
+### How do I authenticate requests to the Token API?
+
+Authentication is managed via JWT tokens obtained through thegraph.market. JWT tokens issued by Pinax are also supported.
+
+### Does the Token API provide a client SDK?
+
+There are currently no plans for a client SDK. We encourage feedback on any SDKs or integrations you would like to see on [Discord](https://discord.gg/graphprotocol).
+
+### Are there plans to support additional blockchains in the future?
+
+Yes, we plan to support more blockchains. Please share your feedback on desired chains on [Discord](https://discord.gg/graphprotocol).
+
+### Are there plans to offer data closer to the chain head?
+
+Yes, improvements to provide data closer to the chain head are planned. Your feedback is welcome on [Discord](https://discord.gg/graphprotocol).
+
+### Are there plans to support additional use cases such as NFTs?
+
+We are actively determining our roadmap for additional use cases, including NFTs. Please provide feedback on specific features you would like prioritized on [Discord](https://discord.gg/graphprotocol).
+
+## MCP / LLM / AI Topics
+
+### Is there a time limit for LLM queries?
+
+Yes. The maximum time limit for LLM queries is 10 seconds.
+
+### Is there a known list of LLMs that work with the API?
+
+Yes. At present, we know that Claude Desktop integrates successfully out of the box with our Token API + MCP server. Beyond that, whether an LLM "works" depends on whether it supports the MCP protocol directly (or has a compatible plugin/adapter).  
+Works out of the box:  
+Claude Desktop.  
+Potentially compatible:  
+Other LLMs that can implement the MCP protocol or use an existing adapter to do so.
+
+### Where can I find the MCP client?
+
+[TBD: URL—confirm if hosted on current GitHub repository or under The Graph protocol.]
+
+## Advanced Topics
+
+### Authentication & Access
+
+- **Q: I'm getting 403/401 errors. What's wrong?**  
+  **A:** Check that you included the `Authorization: Bearer <jwt>` header with the correct, non-expired token. Common issues include forgetting the "Bearer" prefix, using an incorrect token, or omitting the header entirely. Ensure you copied the JWT exactly as provided by The Graph Market.
```

Comment:
Why the different format for Q/A? Did `####` look bad? (It's very possible the spacing was off or something.)